plugins/ci/pkg/ci/tfsec.go

```diff
@@ -99,6 +110,12 @@ func (ci *CIScan) ProcessTerraformPath(terraformPath string) ([]models.TFSecResu
 		}
 		logrus.Debugf("updating filename %q to be relative to the repository: %q", reportProperties.Items[i].Location.FileName, newFileName)
 		reportProperties.Items[i].Location.FileName = newFileName
+		if len(reportProperties.Items[i].RuleID) == 0 {
+			reportProperties.Items[i].RuleID = strings.TrimPrefix(reportProperties.Items[i].LongID, "custom-custom-")
```

Comment:
Who/where is `LongID` set?

Reply:
LongID comes from the tfesc custom check output:

{
                        “rule_id”: “”,
                        “long_id”: “custom-custom-fw_allowed_instance_types”,
                        “rule_description”: “The ec2 instance_type isn’t in the permitted list”,
                        “rule_provider”: “custom”,
                        “rule_service”: “custom”,
                        “impact”: “”,
                        “resolution”: “”,
                        “links”: [
                                “https://aws.amazon.com/ec2/instance-types/ - http://internal.acmecorp.com/standards/aws/s3_buckets.html%”
                        ],
                        “description”: “Custom check failed for resource aws_instance.foo. AWS instance isn’t in the permitted list [‘t2.nano’,‘t2.micro’,‘t2.small’,‘t2.medium’,‘t2.large’,‘t2.xlarge’,‘t2.2xlarge’]“,
                        “severity”: “CRITICAL”,
                        “warning”: false,
                        “status”: 0,
                        “resource”: “aws_instance.foo”,
                        “location”: {
                                “filename”: “/Users/james/git/blog/custom/[custom_check.tf](http://custom_check.tf/)”,
                                “start_line”: 1,
                                “end_line”: 13
                        }
                }

Reply:
I see... cool!

[opt] worth considering not removing `custom-custom-` to prevent event type from clashing

Reply:
custom-custom- it's kind of weird 🤣  but also customer will provide:
code: CUS001 for example and this would become custom-custom-cus001 - could be confusing for like Automation Rule